src/Snorter_Ubuntu-14.04.sh

```diff
@@ -82,7 +102,24 @@ function snort_install() {
 	echo -ne "\n\t${CYAN}[i] INFO:${NOCOLOR} /var/log/snort and /etc/snort created and configurated.\n\n"
 	sudo /usr/local/bin/snort -V
 	echo -ne "\n\t${GREEN}[+] INFO:${NOCOLOR} ${BOLD}SNORT${NOCOLOR} is successfully installed and configurated!"
-
+	
+	#Installing OpenAppID
+	echo -ne "\n\t${CYAN}[i] INFO:${NOCOLOR} Installing ${BOLD}$OpenAppID${NOCOLOR}.\n\n"
+	cd $HOME/snort_src/
+	wget --no-check-certificate -P $HOME/snort_src/ https://snort.org/downloads/openappid/5411 -O snort-openappid.tar.gz
```

Comment:
Will do something like this:

~~~~bash
OPENAPPVERSION=$(curl -s -k https://snort.org/downloads | grep -i "snort-openappid.tar.gz" | grep -oP "\/downloads\/openappid\/\d+")
~~~~ 

The output today (20170721) is:

~~~~bash
jbono@MacBook [~]> echo $OPENAPPVERSION
/downloads/openappid/5759
~~~~